internal/dbmodel/sql/postgres/005_add_migrating_controller_column.up.sql

```diff
@@ -0,0 +1,3 @@
+-- 1_5.sql is a migration that adds a column necessary for model migrations
```

Comment:
i think you should also edit this comments from 1_5 to 005, or just remove it

Reply:
Good point, will do